lib/ansible/modules/yum.py

```diff
@@ -544,6 +544,10 @@ def is_group_env_installed(self, name):
 
         return False
 
+    def is_ovirt_node(self):
+        with open('/etc/os-release') as f:
+            return 'VARIANT_ID="ovirt-node"' in f.read()
```

Comment:
I don't think it's good to have an exception just for ovirt-node, can't we make it more common?

Reply:
The issue occurred on RHVH hosts, I figured it's the best way we can distinguish them from other hosts